front-end/src/app/committee/committee-info/committee-info.component.ts

```diff
@@ -40,12 +41,17 @@ export class CommitteeInfoComponent extends DestroyerComponent implements OnInit
     'custodian_name_full',
   ];
 
+  link: string;
+
   constructor(
     private store: Store,
     private fb: FormBuilder,
     private readonly changeDetectorRef: ChangeDetectorRef,
   ) {
     super();
+    this.link = environment.production
+      ? 'https://webforms.stage.gov/webforms/form1/index.htm'
```

Comment:
can we make the link an environment variable rather than storing both in the component?

Reply:
I was on the fence about making it an environment variable but I thought maybe this was the only place this link would be used, but I do like it better as an environment variable. Updated